content/blog/2023/components.md

```diff
@@ -638,7 +638,7 @@ walking us through the steps:
 
 {{ embed.figure(
   data=[{
-    iframe: 'https://www.youtube.com/embed/R4Ri4ft7bXY',
+    youtube: 'R4Ri4ft7bXY',
     title: 'The Good, The Bad, and The Web Components',
```

Comment:
Unrelated to this PR, but it looks like `title` is ignored here -- we might as well remove it anywhere `embed.figure` is called unless we want to support it.